### PR TITLE
feat: language switcher + Ukrainian translation

### PR DESCRIPTION
**Pull request by MariannaMilovanova**
https://user-images.githubusercontent.com/13865639/155896569-96b5078d-18fa-46c3-9b5d-46f9cc43bc7f.mov

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -3,21 +3,31 @@ import {IntlProvider} from "react-intl";
...
 import messagesRU from '../locales/ru.json'
 import messagesEN from '../locales/en.json'
+import messagesUA from '../locales/uk.json'
 
 import "./style.css"
-import {useEffect, useMemo, useState} from "react";
+import React, {useEffect, useMemo, useState} from "react";
+import LanguageDropdown from "../components/LanguageDropdown";
 
+const LANGUAGES = [{value: 'uk', file: messagesUA}, {value: 'ru', file:messagesRU}, {value: 'en', file: messagesEN}];
```
Comment (artemtam) [inline pages/_app.tsx]: We could change it to a Map/object, so all operations would be simpler and faster:

```typescript
const LANGUAGES = [
  uk: messagesUA,
  ru: messagesRU,
  en: messagesEN,
];
```

Then the `useMemo` would be just:

```typescript
const messages = useMemo(() => LANGUAGES[locale] || LANGUAGES.en, [locale]);
```

And the language auto-detection:

```typescript
const detectedLocale = Object.keys(LANGUAGES).includes(navigatorLanguage) ? navigatorLanguage : 'en';
setLocale(detectedLocale);
```

File: package-lock.json [hunk 2]
```diff
@@ -1,3065 +1,8 @@
 {
   "name": "slava-ukraine",
   "version": "0.1.0",
-  "lockfileVersion": 2,
```
Comment (artemtam) [inline package-lock.json]: I used `npm@8`, you're probably using 6th. Time to update :)

File: pages/style.css [hunk 3]
```diff
@@ -132,4 +132,19 @@ p {
 .social a, .social a:visited {
     margin-top: 4px;
     color: rgba(50, 204, 0, 0.4);
-}
\ No newline at end of file
+}
+
+.Dropdown-root {
```
Comment (artemtam) [inline pages/style.css:137]: Ok for me, but maybe using just native select could have been simpler.

File: package-lock.json (hunk 2 above)
Comment (MariannaMilovanova) [reply package-lock.json]: time to use yarn ;)

File: pages/style.css (hunk 3 above)
Comment (MariannaMilovanova) [reply pages/style.css:137]: difficult to style native simple select to look like this, and react-dropdown quite light lib 

image

**Review by artemtam — APPROVED** [review]
Thank you ❤️

**Pull request merged**